three.js/src/new-api/arjs-session.js

```diff
@@ -88,9 +88,17 @@ const Session = function(parameters){
 	var arContext = _this.arContext = new Context(parameters.contextParameters)
 
 	// initialize it
-	_this.arContext.init(()=>{
-		_this.arContext.arController.orientation = 	getSourceOrientation();
-		_this.arContext.arController.options.orientation = getSourceOrientation();
+	window.addEventListener('arjs-video-loaded', function () {
```

Comment:
i added the event listener, but seems that has no effect. The `domElement.videoWidth` and `domElement.videoHeight` are always 0. Video is not really inited? see https://developer.mozilla.org/en-US/docs/Web/API/HTMLVideoElement/videoWidth#value
Maybe it's needed a delay? Witout this every time we got a wrong orientation (portrait) and so the model is skewed  dued to wrong calculation inside artoolkit5-js

Reply:
encapsulating these lines:
 ```
arContext.arController.orientation = getSourceOrientation();
 arContext.arController.options.orientation = getSourceOrientation()
```
in a setTimeout does not change anything. I will use another approach see commit https://github.com/AR-js-org/AR.js/pull/359/commits/2522fdcf5cf626b6b3c4c9f155de128a8760d948